### PR TITLE
Adds common errors

### PR DESCRIPTION
**Pull request by uF4No**
Identified a few common errors with @StanislavBreadless and added them to the tutorial for reference.
Also increased the amount of ETH we send to the paymaster after it's deployed to avoid issues.
In addition, removes warning on top.

### CODE REVIEW

File: docs/dev/tutorials/custom-paymaster-tutorial.md [hunk 1]
```diff
@@ -400,6 +389,12 @@ Balance of the user before mint: 3
 Balance of the user after mint: 102
 ```
 
+## Common errors
+
+If the `use-paymaster.ts` script fails with error `Failed to submit transaction: Failed to validate the transaction. Reason: Validation revert: Paymaster validation error: Failed to transfer funds to the bootloader`, please try sending additional ETH to the paymaster so it has enough funds to pay for the transaction. You can use the [zkSync portal](https://staging-portal.zksync.dev/).
```
Comment (barakshani) [inline docs/dev/tutorials/custom-paymaster-tutorial.md:394]: Remove staging in the link.

> You can use the [zkSync portal](https://staging-portal.zksync.dev/).

I'd say "You can use zkSync Portal", I think we just refer to it as a name

File: docs/dev/tutorials/custom-paymaster-tutorial.md [hunk 2]
```diff
@@ -400,6 +389,12 @@ Balance of the user before mint: 3
...
+
+If the `use-paymaster.ts` script fails when minting new ERC20 tokens with error `Error: transaction failed`, and the transactions appear with status "Failed" in the [zkSync explorer](https://scan-v2.zksync.dev/), please reach out to us on our Discord. As a workaround, try including the `gasLimit` parameter in the transaction.
```
Comment (barakshani) [inline docs/dev/tutorials/custom-paymaster-tutorial.md:396]: maybe add a direct link to Discord? or at least to the "contact us" page?

File: docs/dev/tutorials/custom-paymaster-tutorial.md (hunk 1 above)
Comment (uF4No) [reply docs/dev/tutorials/custom-paymaster-tutorial.md]: fixed in 3940b2be683846ca0039542730511eb23ec262c4

File: docs/dev/tutorials/custom-paymaster-tutorial.md (hunk 2 above)
Comment (uF4No) [reply docs/dev/tutorials/custom-paymaster-tutorial.md]: fixed in 3940b2be683846ca0039542730511eb23ec262c4

**Review by barakshani — APPROVED** [review]

**Comment by uF4No** [comment]
Merging after @barakshani approval (thanks)🤙

**Pull request merged**